lms/templates/ccx/schedule.html

```diff
@@ -29,22 +29,24 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
+<section id="enter-date-modal" class="modal"
+         tabindex="-1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
     <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
       <span class="sr">
         ${_("Close")}
       </span>
     </button>
     <header>
-      <h2></h2>
+      <h2 id="ccx_schedule_set_date_heading"></h2>
     </header>
     <form role="form">
       <div class="field datepair">
-        <label></label>
-        <input placeholder="Date" class="date" type="text" name="date"/ size="11">
-        <input placeholder="Time" class="time" type="text" name="time"/ size="6">
+        <input placeholder="Date" class="date" type="text" name="date"
```

Comment:
i prefer using explicit labels rather than aria-labels.  See comment below for ideas about mixing offscreen text and on screen text to make a visual and non-visual label that works for all audiences:
https://github.com/edx/edx-platform/pull/9114/files#r36677162